apps/system-e2e/src/support/utils.ts

```diff
@@ -1,97 +1,103 @@
-}
-
-const aliasQuery = (
-  req: CyHttpMessages.IncomingHttpRequest,
-  operationName: string,
-) => {
-  if (hasOperationName(req, operationName)) {
-    req.alias = operationName
-  }
+export enum BaseAuthority {
+  dev = 'beta.dev01.devland.is',
+  staging = 'beta.staging01.devland.is',
+  ads = 'loftbru.dev01.devland.is',
+  prod = 'island.is',
+  local = 'localhost:4200',
 }
 
-const getFakeUser = (fakeUsers: FixtureUser[], name: string): FixtureUser =>
-  fakeUsers
-    .filter((e) => e.name.toLowerCase().includes(name.toLowerCase()))
-    .reduce((e) => e)
-
-const getDiscountUser = (
-  fakeUser: FixtureUser,
-  discounts: ADSUserWithDiscount[],
-) =>
-  discounts.filter((e) => e.nationalId === fakeUser.nationalId).reduce((e) => e)
-
-const getDiscountData = (
-  fakeUser: FixtureUser,
-  res: CyHttpMessages.BaseMessage | undefined,
-) => {
-  const discounts =
-    (res?.body.data.discounts as ADSUserWithDiscount[]) ||
-    cypressError('Error getting response data')
-  return { discounts, user: getDiscountUser(fakeUser, discounts) }
+export enum AuthUrl {
+  dev = 'https://identity-server.dev01.devland.is',
+  staging = 'https://identity-server.staging01.devland.is',
+  prod = 'https://innskra.island.is',
+  local = dev,
 }
 
-const getEnvironmentBaseUrl = (authority: string) => {
-  const baseurlPrefix = process.env.BASE_URL_PREFIX ?? Cypress.env('basePrefix')
+export const getEnvironmentBaseUrl = (authority: string) => {
+  const baseurlPrefix = process.env.BASE_URL_PREFIX ?? ''
   const prefix =
     (baseurlPrefix?.length ?? 0) > 0 && baseurlPrefix !== 'main'
       ? `${baseurlPrefix}-`
       : ''
   return `https://${prefix}${authority}`
 }
-
-const getEnvironmentUrls = (env: TestEnvironment) => {
+export const getEnvironmentUrls = (env: TestEnvironment) => {
   return env === 'dev'
     ? {
         authUrl: AuthUrl.dev,
-        baseUrl: getEnvironmentBaseUrl(BaseAuthority.dev),
+        islandisBaseUrl: getEnvironmentBaseUrl(BaseAuthority.dev),
+        adsBaseUrl: getEnvironmentBaseUrl(BaseAuthority.ads),
       }
     : env === 'prod'
     ? {
         authUrl: AuthUrl.prod,
-        baseUrl: getEnvironmentBaseUrl(BaseAuthority.prod),
+        islandisBaseUrl: getEnvironmentBaseUrl(BaseAuthority.prod),
+        adsBaseUrl: getEnvironmentBaseUrl('https://loftbru.island.is'),
       }
     : env === 'staging'
     ? {
         authUrl: AuthUrl.staging,
-        baseUrl: getEnvironmentBaseUrl(BaseAuthority.staging),
+        islandisBaseUrl: getEnvironmentBaseUrl(BaseAuthority.staging),
+        adsBaseUrl: getEnvironmentBaseUrl(
+          'https://loftbru.staging01.devland.is',
+        ),
       }
-    : { authUrl: AuthUrl.local, baseUrl: `http://${BaseAuthority.local}` }
+    : {
+        authUrl: AuthUrl.local,
+        islandisBaseUrl: `http://${BaseAuthority.local}`,
+        adsBaseUrl: `http://${BaseAuthority.local}`,
+      }
+}
+export type CognitoCreds = {
+  username: string
+  password: string
```

Comment:
This would be more readable as a switch statement/dictionary lookup

Reply:
Definitely, forgot to refactor this.